.github/workflows/revert-pr.yaml

```
@@ -0,0 +1,91 @@
+          PR_NUMBER=$(bash ./scripts/ci/get-pr.sh "$COMMIT_SHA")
+
+          if [ -n "$PR_NUMBER" ]; then
+            echo "PR_NUMBER=$PR_NUMBER" >> $GITHUB_OUTPUT
+          else
+            echo "No merged PR found for commit $COMMIT_SHA."
+          fi
+
+      - name: Add Revert Label
+        id: add-revert-label
+        if: ${{ steps.fetch-pr.outputs.PR_NUMBER && github.event.workflow_run.conclusion == 'failure' }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:1:1: PR_NUMER appears unused. Verify use (or export if used externally)

(shellcheck)